device/globals/exit.gd

```diff
@@ -0,0 +1,44 @@
+extends TextureButton  # XXX: Use TextureButton until background is fixed so CollisionPolygon2D will work
```

Comment:
As discussed on IRC, it might be better if this script shares a common base with `item.gd`, so that less code is duplicated. Eg. a `hotspot.gd` which extends `interactive.gd` and is extended by `item.gd` and this script. Is that in sync with your thoughts as well?

Reply:
Exactly this. Have smaller explicit scripts for these things.

Reply:
I'm force-pushing a version that extends `interactive.gd`, it's clearly the cleaner approach and it works.

It does introduce `Speed`, `Scale on map` and `Light on map` which don't necessarily make sense, unless you want to alter the lighting of an exit under some conditions. Still it's better to expose this stuff, because it draws attention to things that may be relevant to refactor later.

For example, do items require `Speed` if there's going to be an `npc.gd` script?